README.md

```diff
@@ -345,11 +345,14 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
 * [Implement customized TrainingService](docs/en_US/TrainingService/HowToImplementTrainingService.md)
 
 
-## **External Repositories**
-Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
-* Run [ENAS](examples/tuners/enas_nni/README.md) in NNI
-* Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+## **External Repositories and References**
+With authors' permission, we listed a set of NNI usage examples and relevant articles.
+
+* **Repo** - Run [ENAS](examples/tuners/enas_nni/README.md) with NNI
+* **Repo** - Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) with NNI 
+* **Repo** - [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) with NNI 
+* **Repo** - [Hyperparameter Tuning for Matrix Factorization](https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb) with NNI
+* **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison by @gaocegege - 总结与分析 section of design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new)
```

Comment:
If we only care about the last section, we can use 
```suggestion
* **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison by @gaocegege - 总结与分析 section of design and implementation of kubeflow/katib](http://gaocegege.com/Blog/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/katib-new#%E6%80%BB%E7%BB%93%E4%B8%8E%E5%88%86%E6%9E%90)
```

Reply:
Thanks @gaocegege, that's a better fit. @rabbit008 let's use this section link.

Reply:
thanks for @gaocegege ,this pr has been merged. I will request a new branch to fix this link.